.jenkins/pipelines/pr.Jenkinsfile

```diff
@@ -10,12 +10,30 @@ APPROVED_AUTHORS = [
     'RRathna',
     'paulcallen',
     'radhikaj',
-    'rs--',
     'salsal97',
-    'Sahakait',
     'vtikoo'
 ]
 
+// Do not trigger a full CI build if changes contain only these ignored files
+IGNORED_DIRS = [
+    '.jenkins/infrastructure',
+    '.jenkins/docker',
+    '.azure_pipelines',
+    '.github',
+    'doc'
+]
+IGNORED_FILES = [
+    '.gitignore',
+    'arch.png',
+    'BUILDING.md',
+    'CONTRIBUTING.md',
+    'DOCKER_IMAGES.md',
+    'LICENSE',
+    'owners.txt',
+    'README.md',
+    'VERSION'
```

Comment:
Also is it possible to add sample READMEs to this list?

Reply:
Yes, good suggestion, thanks!